tox.ini

```diff
@@ -7,7 +7,7 @@ envlist = flake8,ansible-lint
 basepython = python2.7
 deps =
     ansible>=1.9.1,<2.0.0
-    -ropenstack-ansible/test-requirements.txt
+    -rtest-requirements.txt
```

Comment:
yeah osa kilo has ```dev-requirements.txt```, it was renamed to ```test-requirements.txt``` in osa liberty (see https://github.com/rcbops/rpc-openstack/commit/9192f94da6a6f9b916b8ad294eae5a2ae44d261b)